### DISCUSSION TITLE
Advice on my AI CFD project using pyAnsys

### DISCUSSION THREAD

**Discussion by McCumiskey** [ansys/pyfluent#3136] [General]
My project is to do cfd on buildings with a HVAC system with a circular mass-flow inlet and outlet that moves across the ceiling randomly for around 100 iterations. Ansys would then return the x,y,z position and velocity. An algorithm would then turn the x,y,z position into a function of position relative to the nearest walls and inlet and outlet, this serves as inputs to a Neural Network. I want to do this on PyAnsys or pyMAPDL to streamline the process. Do you have any advice? Also when I run 'session = pyfluent.launch_fluent()' I get an error 'AnsysVersionNotFound: Verify the value of the 'AWP_ROOT' environment variable.' If I have ansys workbench already installed what file should AWP ROOT point to?
Thanks for reading!

**Comment by RobPasMue** [comment]
@ansys/pyfluent-maintainers - some advice?

**Comment by prmukherj** [comment]
AWP root should be like: "AWP_ROOT241=C:\Program Files\ANSYS Inc\v241"

**Comment by hpohekar** [comment]
@RobPasMue  `AWP_ROOT242 = "D:\Installations\Ansys\v242_15072024\ANSYS Inc\v242"`

Path should be set upto `...\ANSYS Inc\v242" in the installation directory.
